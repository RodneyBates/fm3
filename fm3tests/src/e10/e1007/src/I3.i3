INTERFACE I3

; IMPORT       I1 

; END I3 
.

