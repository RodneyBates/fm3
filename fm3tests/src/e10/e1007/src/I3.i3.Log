[31mERROR: [0mI3.i3[3,15] Cyclic import of
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I1.i3, which at [4,11], imports
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I2.i3, which at [6,13], imports
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I3.i3.

