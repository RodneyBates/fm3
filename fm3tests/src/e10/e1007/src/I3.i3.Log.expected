[31mERROR: [0mI3.i3[3,15] Cyclic import of
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I1.i3, which at [4,11], imports
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I2.i3, which at [6,13], imports
       /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1007/src/I3.i3.

[32mINFO:  [0mI3.i3 Pass 1 output file I3.i3.FM3Pass1 has 41 bytes.
[32mINFO:  [0mI3.i3 Patch stack I3.i3.FM3Patch peak size = 19 bytes.
[32mINFO:  [0mI3.i3 Pass 2 output file I3.i3.FM3Pass2 has 40 bytes.
